### DISCUSSION TITLE
有什么问题

### DISCUSSION THREAD

**Discussion by kakajun** [kakajun/vue3-sketch-ruler#21] [Q&A]
来这里讨论

**Comment by 752337625** [comment]
![image](https://github.com/user-attachments/assets/75b7b6f2-9da7-4bdd-be63-23f322ffbb22)
![image](https://github.com/user-attachments/assets/41e9e081-e2eb-4ad2-8cc7-24fbc634c9d7)
当设置为红色框数据后，高度标尺无法显示出来


代码如下


![image](https://github.com/user-attachments/assets/7baa269c-e40b-4636-ac59-1b4667e98e0d)





import { useElementBounding } from '@vueuse/core';
